src/test/ui/liveness/liveness-move-in-loop.stderr

```diff
@@ -4,8 +4,22 @@ error[E0382]: use of moved value: `y`
 LL |     let y: Box<isize> = 42.into();
    |         - move occurs because `y` has type `Box<isize>`, which does not implement the `Copy` trait
 ...
+LL |     loop {
+   |     ---- inside of this loop
+LL |         println!("{}", y);
+LL |         loop {
+   |         ---- inside of this loop
+LL |             loop {
+   |             ---- inside of this loop
+LL |                 loop {
+   |                 ---- inside of this loop
```

Comment:
is it necessary to point out all the loops?

Reply:
I think it is worth it to highlight them. This case is unusual, but every one of these loops will affect the move, and we don't want people to have to compile 4 times when they have code like this to fully understand their problem.